Automattic/jetpack
# My Jetpack: Fix button height in the Interstitial pages

This PR My fixes button height in the Interstitial pages, reducing the scope where these styles were being applied. Before, they affected all buttons present on the page. Now, the height is only applied to the My Jetpack overview page.

Fixes https://github.com/Automattic/jetpack/issues/23187

#### Changes proposed in this Pull Request:

* My Jetpack: Fix button height in the Interstitial pages

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Go to My Jetpack overview page
* Confirm the button height is 32px
<img width="444" alt="image" src="https://user-images.githubusercontent.com/77539/156355732-00a797c0-69e3-454f-b2d5-f6552db2643c.png">

* Go to an interstitial page
* Confirm the button text is properly aligned vertically

before | after
-----|------
<img width="518" alt="image" src="https://user-images.githubusercontent.com/77539/156355989-82bdc09b-4039-463a-928d-522aabecfa0c.png"> | <img width="517" alt="image" src="https://user-images.githubusercontent.com/77539/156355837-f81305af-bf6a-4111-9734-1b49617229dc.png">
